source/usage-examples/updateOne.txt

```diff
@@ -3,3 +3,68 @@ Update a Document
 =================
 
 .. default-domain:: mongodb
+
+You can update a document in a collection by using the ``Collection.UpdateOne()``
```

Comment:
suggestion: I think it's ok to shorten this to just ``UpdateOne()``.

```suggestion
You can update a document in a collection by using the ``UpdateOne()``
```

Reply:
accepted